src/lib/components/workflow/pending-activity/pending-activity-card.svelte

```diff
@@ -188,14 +189,32 @@
         <p class="text-sm text-secondary/80">
           {translate('workflows.last-failure')}
         </p>
-        <CodeBlock
-          content={stringifyWithBigInt(
-            omit(activity.lastFailure, 'stackTrace'),
-          )}
-          maxHeight={384}
-          copyIconTitle={translate('common.copy-icon-title')}
-          copySuccessIconTitle={translate('common.copy-success-icon-title')}
-        />
+        {#if activity.lastFailure?.encodedAttributes}
+          {#key activity.attempt}
```

Comment:
Shouldn't need this with [this change](https://github.com/temporalio/ui/pull/2825/files#diff-f5df298e3cf10bab6f20ce354a1c5e425bf2f7e174a88ca006a11fd0e21d8f83R33), but can update in my PR.

Reply:
👍